### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by derrickdso**

### CODE REVIEW

File: wp/wp-content/plugins/phila.gov-customization/public/class-phila-gov-site-wide-alert-rendering.php [hunk 1]
```diff
@@ -76,8 +76,31 @@ function dateTimeFormat($date) {
...
+                  <div class="cell medium-auto medium-shrink small-24 align-self-middle">
+                    <i class="fas fa-check-to-slot fa-fw fa-2x icon hide-for-small-only" aria-hidden="true"></i>
+                    <div class="voting-text">
+                      <p class="mbn"><b>Election day is Nov. 8, 2022</b></p>
+                      <p class="vote-deadline mbn">The deadline to register to vote is Oct. 24, 2022.</p>
+                      </div>
+                    </div>
+                    <div class="cell medium-auto medium-shrink small-24 align-self-right">
+                    <div class="vote-button">
+                    <a class="button" href="https://vote.phila.gov/my-vote-my-way/"><b>MAKE A PLAN TO VOTE</b></a>
```
Comment (karissademi) [inline wp/wp-content/plugins/phila.gov-customization/public/class-phila-gov-site-wide-alert-rendering.php:95]: We can drop the <b> and the all caps text here. Those are already part of the button styles, so no need to hard code it that way.

Comment (karissademi) [inline wp/wp-content/plugins/phila.gov-customization/public/class-phila-gov-site-wide-alert-rendering.php:95]: At some point, this link changed and I didn't update the ticket, apologies: https://vote.phila.gov/voting/my-vote-my-way/

File: wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss [hunk 2]
```diff
@@ -26,4 +26,36 @@
       text-decoration: underline;
     }
   }
+}
+.voting-banner {
+  min-height: 60px;
+  background-color: #f0f0f0;
```
Comment (karissademi) [inline wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss:32]: We can use `background-color: color(ghost-gray)` here. See: https://standards.phila.gov/docs/design/brand-elements/color-palette.html

File: wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss [hunk 3]
```diff
@@ -26,4 +26,36 @@
...
+  .vote-button{
+    color: black;
```
Comment (karissademi) [inline wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss:34]: Can we double check that all these rules are needed? For example, the color, height and text-alignment are all likely already part of button styles. 

Same comment for the float: left on the link. Are we sure that's necessary?

**Review by karissademi — CHANGES_REQUESTED** [review]
A couple of comments, mostly around double checking that all the styles are necessary. This is great work!

File: wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss (hunk 3 above)
Comment (derrickdso) [reply wp/wp-content/themes/phila.gov-theme/css/scss/_alerts.scss]: float left is to left align the whole button on mobile but the extra styles were cleaned up

**Review by karissademi — APPROVED** [review]

**Pull request merged**